ts/components/messages/MessageAgenda.tsx

```diff
@@ -291,6 +315,9 @@ class MessageAgenda extends React.PureComponent<Props, State> {
   }
 
   private loadMoreData() {
+    this.setState({
```

Comment:
```suggestion
    this.setState(
      {
        isLoading: true
      },
      this.props.onMoreDataRequest
    );
```
call _this.props.onMoreDataRequest_ when the state will be updated

Reply:
This state (now renamed _isLoadingProgress_) is used for a progress check and a delay in the disappearance, there is another prop (isRefreshing) connected directly to _this.props.onMoreDataRequest_ to check the status of the pot